### ISSUE TITLE
Cannot open a second editor in another desktop to the same folder

### ISSUE THREAD

**Issue by afchacon** [microsoft/vscode#95390]
Issue Type: <b>Bug</b>
## Steps:
* Open VSCode to a particular folder
* Switch to a new Windows desktop with Ctrl+Win+Right Arrow
* Open VSCode to the same folder as the first step
---
### Expected: 
Folder opens in the current VSCode window

---
### Actual:
Focus goes back to Desktop 1 with the originally opened folder on the first step. And folder is not opened in second instance of VSCode

---

VS Code version: Code 1.44.0 (2aae1f26c72891c399f860409176fe435a154b13, 2020-04-07T23:31:18.860Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-1680 v4 @ 3.40GHz (16 x 3392)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.90GB (53.55GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-swift-syntax|dun|0.0.4
vscode-diff|fab|1.4.0
gc-excelviewer|Gra|2.1.34
swift|Kas|0.2.0
csharp|ms-|1.21.17
vscode-edge-devtools|ms-|1.0.7
python|ms-|2020.3.71659
remote-wsl|ms-|0.44.2
cmake-tools|ms-|1.3.1
cpptools|ms-|0.27.0
debugger-for-chrome|msj|4.12.6
djinni-syntax-highlighting|pat|1.0.3
cmake|twx|0.0.17

</details>

**Comment by yume-chan** [comment]
Duplicate of #10121

TL;DR, use the `Workspaces: Duplicate Workspace in New Window` command.

**Comment by afchacon** [comment]
@yume-chan @chrmarti @JacksonKearl 
While @yume-chan 's answer does work for this situation, the experience is less than ideal.
There was no way for me to know that that was an option aside from word of mouth or opening this ticket as the option is hidden deep within the bowels of the grab bag of possible key binds and not available in any menu.

I'm sure there can be a better solution to this problem than the current one making it a bit more user friendly. Do you think a feature request for this could be put in?

**Comment by chrmarti** [comment]
@afchacon https://github.com/microsoft/vscode/issues/2686 should capture this.
